.github/workflows/test.yml

```diff
@@ -45,28 +41,41 @@ jobs:
         uses: ./.github/actions/bump-skip
         with:
           event-name: ${{ github.event_name }}
-      - name: Pull Vault image
+      - name: Render secrets for tests
```

Comment:
No longer call `local-dev/render-config.sh` from the github action; instead rely on github secrets populated by https://github.com/broadinstitute/terraform-ap-deployments/pull/1742